docs/vsphere_ci/platform-none/README.md

```diff
@@ -0,0 +1,35 @@
+# Adding PTR records for platform=none CI clusters in vSphere
+
+WMCO' CSRs approver requires reverse DNS lookup for each Windows instance that 
+joins the cluster as a Windows worker. For CI clusters with platform-agnostic
+infrastructure (platform=none) a fixed lease pool is configured to ensure latter 
+requirement is met.
+
+The `create-ptr-records.sh` script enables the reverse DNS lookup by creating
+PTR records for each IP address available in the selected subnets.
+
+vSphere CI (`vmc-ci.devcluster.openshift.com`) uses AWS Route53 for DNS
+resolution and the following ci-segments are reserved for platform=none clusters
+- ci-segment-56
+- ci-segment-47
```

Comment:
I think you meant c-segment-57

Reply:
Good catch. Addressed in [79db264](https://github.com/openshift/windows-machine-config-operator/commit/79db264de387be5152554ede15e1a44552609b42)